ansible/files/runtimes.json

```diff
@@ -103,11 +103,19 @@
         "php": [
             {
                 "kind": "php:7.1",
-                "default": true,
+                "default": false,
                 "deprecated": false,
                 "image": {
                     "name": "action-php-v7.1"
                 }
+            },
+            {
+                "kind": "php:7.2",
+                "default": true,
+                "deprecated": false,
+                "image": {
+                    "name": "action-php-v7.2"
```

Comment:
due to #3680, this should specify the prefix as openwhisk and the tag as latest.